commands_function.py

```diff
@@ -0,0 +1,59 @@
+import os
+from abc import ABCMeta, abstractstaticmethod
+import json
+from keys import keys
+
+class Icommand(metaclass=ABCMeta):
+    @abstractstaticmethod
+    def excute():
+        """the interface Icommand"""
+
+class commandfactory:
+    @staticmethod
+    def build_command(args):
+        command_type=args.command
+        if command_type == "create":
+            return create(args.schema)
+        elif command_type == "delete":
+            return delete()
+        elif command_type == "get":
+            return get()
+        elif command_type == "set":
+            return set()
+        
+class create(Icommand):
+    def __init__(self,schema):
+        self.schema=schema
+    def excute(self):
+        schema=self.schema
+        f=open(schema,"r")
+        schima=json.load(f)
```

Comment:
schema not schima 

What happend if the file doesn't exists ? 

I would recommend create a private method here, this method takes a schema_file and return schema object